### PR TITLE
Query & Update animals table

### PR DESCRIPTION
**Pull request by eric2umeh**
In this PR,

- [x] Use database transactions.
- [x] Modify and delete data in SQL by;
- Add SQL to modify the "animals" table with a new species column in `schema.sql`.
- Add SQL to insert more data into the "animals" table in `data.sql`.
- [x] Prepare complex queries that answer analytical [questions](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md).
- Add queries using transactions & add queries using aggregate functions in `queries.sql`.
- [x] Follow correct Gitflow
- [x] Check for Linters

Below are the screenshots of the results of the SQL commands used in this milestone.

1. Add a new column species to the "animals" table & insert data.
Screenshot 2023-03-28 at 17 46 14

- 2.1. Transaction update the animals table by setting the species column to unspecified
Screenshot 2023-03-28 at 17 47 27

- 2.2. Re-update Transactions & commit
Screenshot 2023-03-28 at 17 48 25

- 2.3. Including Queries before COMMIT;
Screenshot 2023-03-29 at 09 12 26

- 3. Transaction delete all records in the animals table, then roll back the transaction.
Screenshot 2023-03-28 at 17 53 15

- 4.1. CRUD transaction
Screenshot 2023-03-28 at 18 06 39

- 4.2. Cont'd
Screenshot 2023-03-28 at 18 06 52

- 5.1. Query Questions
Screenshot 2023-03-28 at 18 34 08

- 5.2.
Screenshot 2023-03-28 at 18 36 17

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +9,64 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+BEGIN WORK;
+UPDATE animals
+SET species='digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species='pokemon'
+WHERE species IS NULL;
+COMMIT WORK;
+SELECT * FROM animals;
```
Comment (AnselemOdims) [inline queries.sql]: - [x] Please kindly include a query to verify that the change was made before committing.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @eric2umeh , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (eric2umeh) [reply queries.sql]: Thank you @AnselemOdims ,
I have included the queries before committing.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @eric2umeh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to make the change requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**